easyrecyclerview/src/main/java/com/jude/easyrecyclerview/EasyRecyclerView.java

```diff
@@ -380,11 +386,44 @@ public void setRefreshingColor(int... col) {
      *
      * @param listener
      */
+
+    /**
+     * Set scroll listener to the recycler
+     *
+     * @deprecated Use {@link #addOnScrollListener(RecyclerView.OnScrollListener)} instead.
+     */
+    @Deprecated
```

Comment:
maybe you can just add the listener to the list here and get rid of `mExternalOnScrollListener`?